README.md

```diff
@@ -47,9 +47,7 @@ The module comes with the following commands:
 
     * There should be no need for such a function, but it is currently required because PowerShell's built-in argument passing is still broken as of PowerShell 7.0, [as summarized in GitHub issue #1995](https://github.com/PowerShell/PowerShell/issues/1995#issuecomment-562334606); should the problem be fixed in a future version, this function will detect the fix and will no longer apply its workarounds.
 
-    * For technical reasons:
-      * Passing `--` is invariably removed by PowerShell; to pass `--` through to the target executable, specify it _twice_.
-      * You must check only `$LASTEXITCODE` for being nonzero in order to determine if the executable signaled failure; do not use `$?`, which always ends up `$true`. Unfortunately, this means that you cannot meaningfully use this function with `&&` and `||`, the [pipeline-chain operators](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_Pipeline_Chain_Operators); however, if _aborting_ your script in case of a nonzero exit code is desired, you can use the `iee` function: see below.
+    * For technical reasons, you must check only `$LASTEXITCODE` for being nonzero in order to determine if the executable signaled failure; do not use `$?`, which always ends up `$true`. Unfortunately, this means that you cannot meaningfully use this function with `&&` and `||`, the [pipeline-chain operators](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_Pipeline_Chain_Operators); however, if _aborting_ your script in case of a nonzero exit code is desired, you can use the `iee` function: see below.
```

Comment:
`ie` is a separate command from `ins`, so the `--` issue should be mentioned here again, framed slightly differently, so we should leave this paragraph as is. (`ins` has a legitimate reason to support `--`, given that it has its own parameters, `ie`, which should just be a conduit for all arguments, does not).